spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogDTO.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.thymeleaf.blog;
```

Comment:
just a thought, could we use lombok here?

Reply:
We could definitely use it here, but it wasn't already imported the module and I wanted to keep the example code as much aligned to the article as possible.